descriptor/qsarmolecular/src/main/java/org/openscience/cdk/geometry/surface/NumericalSurface.java

```diff
@@ -318,4 +318,8 @@ private void translatePoints(int atmIdx, Point3d[][] points, int pointDensity, I
         }
         return (ret);
     }
+
+    public static void main(String[] args) {
+        
+    }
```

Comment:
Needed? Can this be removed?

Reply:
No, it slipped in by mistake, but not sure how to get rid of this

Reply:
just add a commit to remove it? With history rewrites you can really clean it up, but that's not really needed, I think...

Reply:
Ah ok - I was trying to be a bit cleaner, but this is easier.